🀄 Completed preliminary Chinese translation

BytinYarota opened the pull request:
(Except the salt name part actually

BytinYarota committed:
Update zh_cn.json

BytinYarota committed:
翻译reaction前面小部分

怎么这么多啊啊啊啊，先摆了

BytinYarota committed:
推进reaction翻译

BytinYarota committed:
完成reaction部分的翻译

芜湖！

BytinYarota committed:
Create zh_cn.json in salt_name_overrides

BytinYarota committed:
Update added keys in zh_cn

BytinYarota commented:
The current work was mainly done by me, @YuriMontana and @yaodaosang collaboratively, with the help of other members of @CFPAOrg and some other helpful players.
Translating your mod was surely a challenge, with puns and technical terms tricky to deal with. I'm glad that we have came through all these, successfully completing the first stage of work.
The translation is still of poor quality, and we still have a long way to perfection. However, the current fruit definitely sets up a milestone, which we could be proud of.
Last but not least, sincere thanks to you for such an awesome mod, as well as immediate bug fixes, clear explanations and so on. Wish Destroy mod achieves its deserving success!🎉

yaodaosang committed:
Update zh_cn.json

Use Chinese punctuation

Catchmons commented on src/main/resources/assets/destroy/lang/zh_cn.json:
```diff
@@ -156,269 +156,283 @@
-    "destroy.chemical.chloroaurate.iupac": "Tetrachloroaurate (III)",
-    "destroy.chemical.chloroauric_acid": "Chloroauric Acid",
-    "destroy.chemical.chloroauric_acid.iupac": "Tetrachloroauric (III) Acid",
+    "destroy.chemical.acetamide": "乙酰胺",
```
普通名字的话会不会用醋酸会好一点，系统命名法再用乙酸

BytinYarota replied:
同意，另一处也是改的醋酸
以及把你add collaborators了

BytinYarota replied:
已修改√

BytinYarota committed:
🔧乙酸→醋酸&小修小补

将乙酸某的俗称改为醋酸某
将如某某水解的反应名中间加入的

BytinYarota committed:
改进chemistry部分

优化语序、更正链接、统一用词
至L400

yaodaosang committed:
Update zh_cn.json

查缺补漏

yaodaosang committed:
Update zh_cn.json

BytinYarota committed:
完成chemistry的初步改进

BytinYarota committed:
destrot→destroy

"destrot.chemical.unknown"→"destroy.chemical.unknown"
Seems to be a misspelling.

BytinYarota committed:
destrot→destroy

"destrot.chemical.unknown"→"destroy.chemical.unknown" Seems to be a misspelling.

yaodaosang committed:
Merge branch '1.20.1' into helloworld069-patch-1

yaodaosang committed:
Merge pull request #1 from helloworld069/helloworld069-patch-1

完成chemistry的初步改进

BytinYarota committed:
统一为dynamo

BytinYarota committed:
改进882行并列谓语的标点

BytinYarota committed:
修正882行缺漏的逗号

petrolpark commented:
Splendid! Thank you so much for all of your help. If any of you who want the translator Badge don't have it yet, DM me on Discord at `petrolpark`. Once again, thank you!
